contracts/contracts/Sapphire.sol

```diff
@@ -52,7 +54,9 @@ library Sapphire {
         // Sr25519 signature over the provided message.
         Sr25519,
         // Secp256r1 signature over the provided SHA-256 digest.
-        Secp256r1PrehashedSha256
+        Secp256r1PrehashedSha256,
+        // Secp384r1 signature over the provided SHA-384 digest.
```

Comment:
I'm not sure why this description and of other SigningAlg values is not shown at https://api.docs.oasis.io/sol/sapphire-contracts/contracts/Sapphire.sol/library.Sapphire.html#signingalg

Can you use `@dev` tag instead?

Reply:
Annotating individual enum values with NatSpec comments doesn't seem to make any difference to the docs generated by `forge` :(

However, it is possible to annotate the enum itself, e.g.:
```solidity
/// Docs Go here
enum SigningAlg { ... }
```